src/game/turn.ts

```diff
@@ -32,8 +32,30 @@ function getRandomInteger(min: number, max: number): number {
   return Math.floor(Math.random() * (max - min) + min);
 }
 
-export function getRandomSpectrum(): Spectrum {
-  return spectrumData[getRandomInteger(0, 60)];
+export function getRandomSpectrum(spectrumHistory: Array<Spectrum>): Spectrum {
+  // TODO create unit tests to check which of these two are most opitmal.
+
+  // let newSpectrumData = spectrumData;
+
+  // spectrumHistory.map((spectrum) => {
+  //   newSpectrumData = newSpectrumData.filter(
+  //     (dataSpectrum) => dataSpectrum !== spectrum
+  //   );
+  // });
+
+  // const currentRandomIndex = getRandomInteger(0, newSpectrumData.length);
+
+  // return newSpectrumData[currentRandomIndex];
```

Comment:
can you remove this comment? If you need to see the old code again for some reason, you can look at the history of this PR.

Reply:
ok
done it